docs/organizations/using-a-coding-standard.md

```diff
@@ -0,0 +1,95 @@
+# Using a coding standard
+
+Create a coding standard on your organization to define and apply global tool and code pattern configurations consistently across your repositories.
+
+<!--NOTE
+    Add this sentence to the paragraph above when CY-5181 is complete.
+
+You can also apply the coding standard to new repositories automatically by defining the coding standard as default.
+ --->
+
+The organization coding standard helps you ensure that Codacy analyzes all your repositories with the same tool and code pattern settings. For example, use the organization coding standard to ensure that all your repositories and teams are following the same global security rules or coding conventions.
```

Comment:
`... analyzes all your repositories with ...` - seems to tell users that the standard is a set of configs for analysis that will necessarily impact ALL repos.  

should we consider instead of saying this, to say something like `.. analyzes a subset (or all) your repositories with ...`

I do understand that we may not want this to specified so early in the description of the feature to avoid confusing users though, so take this as a suggestion, I'm cool with proceeding with the current version if you find its more appropriate

Reply:
Yes, indeed I was purposefully simplifying here but I understand that this could lead users to think that the coding standard can only apply to all repositories.

I'm going to rewrite this to make it explicit that the users can actually apply the coding standard to the repositories that they choose. :+1:

Reply:
To keep the sentence simple, I decided to change "all your repositories" to "multiple repositories". The example that follows the sentence also helps make it clear that users can apply the coding standard to all repositories if they wish to do so.